templates/macros/clients.html

```diff
@@ -1,8 +1,8 @@
 {% macro classes(client) %}
-{% if client.packages.apple_app_store.app_id and client.packages.apple_app_store.org %}
+{% if client.packages.other_ios_link or client.packages.apple_app_store.app_id and client.packages.apple_app_store.org %}
```

Comment:
If you allow me to be so pedantic, I would love to order this consistently with how the other platforms work ;)

```suggestion
{% if client.packages.apple_app_store.app_id and client.packages.apple_app_store.org or client.packages.other_ios_link %}
```